### ISSUE TITLE
No webcam from client in Ubuntu

### ISSUE THREAD

**Issue by dpetroff** [neutrinolabs/xrdp#2324]
I have an Ubuntu 20.04.3 VM on a Windows 10 host. The option to pass through audio and the integrated webcam is ticked ("Other supported RemoteFX USB devices: Integrated Camera"). Audio (both recording and playback) work (only after I do `pulseaudio -k` in the console, which is inconvenient but ok) but the camera does not. It's simply not listed as a device in Ubuntu. I've been unable to find a permanent solution for the `pulseaudio -k` problem, and without running that command, the audio also acts as though devices do not exist, so I can only assume it's somehow related. Any advice for what to poke?

**Comment by matt335672** [comment]
Duplicate of #2187 

At the moment we don't support this, I'm afraid.
